Common/Tests/Helper/MockHyperwalletInsights.swift

```diff
@@ -0,0 +1,27 @@
+import Insights
+
+public final class MockHyperwalletInsights: HyperwalletInsightsProtocol {
+    public var didTrackClick = false
+    public var didTrackImpression = false
+    public var didTrackError = false
+
+    public init() { }
+
+    public func trackClick(pageName: String, pageGroup: String, link: String, params: [String: String]) {
+        didTrackClick = true
+    }
+
+    public func trackImpression(pageName: String, pageGroup: String, params: [String: String]) {
+        didTrackImpression = true
+    }
+
+    public func trackError(pageName: String, pageGroup: String, errorInfo: ErrorInfo) {
+        didTrackError = true
+    }
+
+    public func resetStates() {
+        didTrackClick = false
+        didTrackImpression = false
+        didTrackError = false
+    }
+}
```

Comment:
We dont need this file

Reply:
where did you find it?

Reply:
Apologies - maybe github was not updating this - I checked out your branch and it looked fine